### ISSUE TITLE
Preview the font

### ISSUE THREAD

**Issue by lainsce** [aimproxy/dotfonts#1]
First issue, but a very important one:

The app should have a preview of the font chosen from the sidebar, above the code snippets. Would help the user to preview what each font looks.

**Comment by aimproxy** [comment]
Yeah, I allready have this in mind, I just need to figure out how to preview the font without need install in the system! I think they can be imported on css and define a font-family! Another Idea how to preview that?

**Comment by lainsce** [comment]
Maybe cache the ttf that's in the Google Fonts API, and if the user changes fonts, that cache is deleted and a new one is made.

**Comment by aimproxy** [comment]
Thanks good idea!

**Comment by saidbakr** [comment]
> Yeah, I allready have this in mind, I just need to figure out how to preview the font without need install in the system! I think they can be imported on css and define a font-family! Another Idea how to preview that?

Simply, you may embed web page component in the preview. However and regarding less the implementation of the preview, **it should** support preview with custom text entered by the user and that text should be controlled in size by the user too.
